packages/ses/src/tame-faux-data-properties.js

```diff
@@ -0,0 +1,131 @@
+import {
+  getOwnPropertyDescriptor,
+  apply,
+  defineProperty,
+  toStringTagSymbol,
+} from './commons.js';
+
+const throws = thunk => {
+  try {
+    thunk();
+  } catch (er) {
+    return true;
+  }
+  return false;
+};
+
+/**
+ * Exported for convenience of unit testing. Harmless, but not expected
+ * to be useful by itself.
+ *
+ * @param {any} obj
+ * @param {string|symbol} prop
+ * @param {any} expectedValue
+ * @returns {boolean}
+ */
+export const tameFauxDataProperty = (obj, prop, expectedValue) => {
+  if (obj === undefined) {
+    return false;
+  }
+  const desc = getOwnPropertyDescriptor(obj, prop);
+  if (!desc || 'value' in desc) {
+    return false;
+  }
+  const { get, set } = desc;
+  if (typeof get !== 'function') {
+    return false;
+  }
+  if (typeof set !== 'function') {
+    return false;
+  }
+  const observedValue = get();
+  if (observedValue !== expectedValue) {
+    return false;
+  }
+  if (apply(get, obj, []) !== expectedValue) {
+    return false;
+  }
```

Comment:
This feels redundant. I would be fine just testing the second one.

Reply:
Done

Reply:
In light of how this thread has gone, I added it back in. But I added a comment to each to explain what it tests.